openmdao/recorders/case.py

```diff
@@ -0,0 +1,65 @@
+class Case(object):
+    """ Case wraps the data from a single iteration/case of a recording
+    to make it more easily accessible to the user.
+
+    Attributes
+    ----------
+    filename : str
+        The file from which the case was loaded
+    case_id : str
+        The identifier of the case/iteration in the case recorder
+    timestamp : str
+        Time of execution of the case
+    success : str
+        Success flag for the case
+    msg : str
+        Message associated with the case
+    parameters : dict
+        Parameters in the case.  Keyed by parameter path name, values are
+        float or dict.
+    unknowns : dict
+        Unknowns in the case.  Keyed by variable path name, values are
+        float or dict.
+    derivs : dict
+        Derivatives in the case.  Different optimizers store the derivatives
+        in different ways.
+    resids : dict
+        Residuals in the case.  Keyed by variable path name, values are
+        float or dict.
+
+
+
+    """
+
+    def __init__(self, filename, case_id, case_dict):
```

Comment:
no doctsring for the constructor?  I think we have been documenting the `__init__` Args in the class level docstring